rest-api/dao/physical_measurements_dao.py

```diff
@@ -370,11 +371,16 @@ def _do_update_with_patch(self, session, measurement, resource):
       measurement.cancelledSiteId = site_id
       measurement.cancelledTime = clock.CLOCK.now()
       measurement.status = PhysicalMeasurementsStatus.CANCELLED
+      measurement.finalized = None
```

Comment:
Should finalizedSiteId / finalizedUsername also be cleared here?

Reply:
The problem with physical measurements is they have no history like other tables, so I don't believe we can get them back if the order is restored later. Unless we want to take the approach that who ever restores it, becomes the finalized site/author, which I considered. Thoughts ?

Reply:
Got it - what you are describing sounds reasonable. Checking though - is that the current behavior in this PR?

When a restore happens - won't it overwrite the finalized{userName,siteId} at L381 ? Or is that somehow getting pulled out of the database?

Reply:
you're right, they will be overwritten by current site_id/author of the request. Maybe we should rethink that logic later, but at least for now, the values are not None for cancelled.